dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -88,6 +88,46 @@ def funometer_by_script_level
     end
   end
 
+  def level_answers
+    authorize! :read, :reports
+
+    @headers = ['Level ID', 'User Email', 'Data']
+    @responses = {}
+    @response_limit = 100
+    if params[:levels] || params[:mc_levels]
```

Comment:
could you just make one param? It looks like you just smash them together. You can ask a Level whether it is a multi if you load its object.